File: ScannerVB/Model/RefComp.vb
Public Class RefComp

    Enum LeType As Byte
        Ocx = 0
        Ref = 1
    End Enum

    Property NomApplication As String
    Property NomDuProjet As String
    Property MotClef As LeType
    Property NomDuFichierRefOCxTlb As String
    Property libelleRefOCxTlb As String
    Property ExtraitDeLigneVBP As String

End Class

File: ScannerVB6/ScannerVB6/Model/RefComp.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ScannerVB6.Model
{
    public class RefComp
    {
         public enum LeType : byte
        {
            Ocx = 0,
            Ref = 1
        }

        public required string NomApplication { get; set; }
        public required string NomDuProjet { get; set; }
        public LeType MotClef { get; set; }
        public string? NomDuFichierRefOCxTlb { get; set; }
        public string? libelleRefOCxTlb { get; set; }
        public string? ExtraitDeLigneVBP { get; set; }

    }
}
